src/HaskellCI/GitHub.hs

```diff
@@ -556,16 +561,37 @@ cat path contents = sh $ concat
     , "EOF"
     ]
 
+-- |
+--
+-- >>> parseGitHubRepo "git@github.com:haskell-CI/haskell-ci.git"
+-- Just "haskell-CI/haskell-ci"
+--
+-- >>> parseGitHubRepo "git@github.com:haskell-CI/haskell-ci"
+-- Just "haskell-CI/haskell-ci"
+--
+-- >>> parseGitHubRepo "https://github.com/haskell-CI/haskell-ci.git"
+-- Just "haskell-CI/haskell-ci"
+--
+-- >>> parseGitHubRepo "https://github.com/haskell-CI/haskell-ci"
+-- Just "haskell-CI/haskell-ci"
+--
 parseGitHubRepo :: Text -> Maybe Text
 parseGitHubRepo t =
     either (const Nothing) Just $ Atto.parseOnly (parser <* Atto.endOfInput) t
   where
     parser :: Atto.Parser Text
-    parser = sshP
+    parser = sshP <|> httpsP
 
     sshP :: Atto.Parser Text
     sshP = do
         _ <- Atto.string "git@github.com:"
         repo <- Atto.takeWhile (/= '.')
-        _ <- Atto.string ".git"
+        _ <- optional (Atto.string ".git")
+        return repo
+
+    httpsP :: Atto.Parser Text
+    httpsP = do
+        _ <- Atto.string "https://github.com/"
```

Comment:
Speaking of common use cases, I think it might be worthwhile to accept `git://` as a possible prefix, if only because the official Git documentation [uses that as an example](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes).